linera-witty-macros/src/wit_export/function_information.rs

```diff
@@ -281,7 +281,7 @@ fn path_matches_segments(path: &Path, segments: &[&str]) -> bool {
     }
 
     for (index, (segment, expected)) in path.segments.iter().zip(segments).enumerate() {
-        let with_type_parameters = index == segments.len();
+        let with_type_parameters = index == segments.len() - 1;
```

Comment:
This is the bug fix.

Reply:
Usually I would say that the loop is begging to be unrolled but ok :)